cumulusci/core/keychain/encrypted_file_project_keychain.py

```diff
@@ -33,7 +35,7 @@ def _load_files(self, dirname, extension, key):
                 name = item.replace(extension, "")
                 if key not in self.config:
                     self.config[key] = {}
-                self.config[key][name] = config
+                self.config[key][name] = constructor(config)
```

Comment:
I would make constructor default to None, and then do

```suggestion
                self.config[key][name] = constructor(config) if constructor else config
```